python/sparkdl/transformers/named_image.py

```diff
@@ -211,13 +211,15 @@ def _transform(self, dataset):
         modelGraphSpec = _buildTFGraphForName(self.getModelName(), self.getFeaturize())
         inputCol = self.getInputCol()
         resizedCol = "__sdl_imagesResized"
-        tfTransformer = TFImageTransformer(inputCol=resizedCol,
-                                           outputCol=self.getOutputCol(),
-                                           graph=modelGraphSpec["graph"],
-                                           inputTensor=modelGraphSpec["inputTensorName"],
-                                           outputTensor=modelGraphSpec["outputTensorName"],
-                                           outputMode=modelGraphSpec["outputMode"])
-        resizeUdf = resizeImage(modelGraphSpec["inputTensorSize"])
+        tfTransformer = TFImageTransformer(
```

Comment:
the previous styling was correct

Reply:
it's still under 100 characters though. there are definitely longer lines in the file.